src/utils/tagged-template-literal.js

```diff
@@ -141,6 +141,17 @@ const interleave = (quasis, expressions, absolutePath) => {
        */
       substitute = '$dummyValue'
     }
+    // Make sure substituted by same count of lines
+    const intentCount = prevText.split('\n').pop().length
+    const intent = new Array(intentCount).fill(' ').join('')
+    const targetLines = quasis[i + 1].loc.start.line - quasis[i].loc.end.line + 1
+    let currentLines = substitute.split('\n').length
+    while (currentLines < targetLines) {
```

Comment:
Okay, so just to make sure I understand what you're doing here, you're just making sure that we substitute the correct amount of lines in so we don't have propagating line errors further down, yes? As in, we make sure that the length of our processed CSS is the same as the length of their actual TTL (Tagged Template Literal). Right?

Reply:
Right. But it's a little hard to keep line counts without introducing new styling errors. How about padding lines by `/* stylelint-disable-line */`?

Reply:
Hey, that's quite a good idea. I like it. It would make the logic so much easier, just add x empty lines that only say `/* stylelint-disable-line */` and we'll be golden, right?

Make sure there aren't any edge cases with the indentation rules / empty lines before blocks etc. that are introduced as I'm not sure exactly how stylelint works with the indentation rules, but yeah I like that idea, the simpler the better!